examples/layer-browser/src/app.js

```diff
@@ -197,38 +207,46 @@ class App extends PureComponent {
   /* eslint-enable max-depth */
 
   _getModelMatrix(index, coordinateSystem) {
-    // the rotation controls works only for layers in
-    // meter offset projection mode. They are commented out
-    // here since layer browser currently only have one layer
-    // in this mode.
     const {settings: {separation}} = this.state;
     const modelMatrix = new Matrix4().translate([0, 0, 1000 * index * separation]);
 
-    // if (coordinateSystem === COORDINATE_SYSTEM.METER_OFFSETS) {
-    //   const {settings: {rotationZ, rotationX}} = this.state;
-    //   modelMatrix.rotateZ(index * rotationZ * Math.PI);
-    //   modelMatrix.rotateX(index * rotationX * Math.PI);
-    // }
+    switch (coordinateSystem) {
+    case COORDINATE_SYSTEM.METER_OFFSETS:
+    case COORDINATE_SYSTEM.IDENTITY:
+      const {settings: {rotationZ, rotationX}} = this.state;
+      modelMatrix.rotateZ(index * rotationZ * Math.PI);
+      modelMatrix.rotateX(index * rotationX * Math.PI);
+      break;
+    default:
+      // Rotations don't work well for layers in lng lat coordinates
+      // since the origin is far away.
+      // We could rotate around current view point...
+    }
 
     return modelMatrix;
   }
 
-  _renderNoTokenWarning() {
-    /* eslint-disable max-len */
-    return (
-      <div id="no-token-warning">
-        <h3>No Mapbox access token found.</h3>
-        Read <a href="http://uber.github.io/deck.gl/#/documentation/overview/getting-started#note-on-map-tokens-">"Note on Map Tokens"</a> for information on setting up your basemap.
-      </div>
-    );
-    /* eslint-disable max-len */
```

Comment:
why remove token warning message? we need it for geospatial examples.

Reply:
> why remove token warning message? we need it for geospatial examples.

The good news is that react-map-gl 3.1 now handles token warning, no need to do it in each example.